nucleus/model.py

```diff
@@ -343,3 +345,10 @@ def remove_trained_slice_ids(self, slide_ids: List[str]):
             )
 
         return response.json()
+    def ungrouped_export(self, model_run_id):
+        json_response = self._client.make_request(
+            payload={},
+            route=f"modelRun/{model_run_id}/ungrouped",
+            requests_command=requests.get,
+        )
+        return format_prediction_response({ANNOTATIONS_KEY: json_response})
```

Comment:
Lets name this something a bit better. I'm not sure why the endpoint was originally named ungrouped, but this method could be called export_by_model_run_id or something like that